### ISSUE TITLE
About the code

### ISSUE THREAD

**Issue by dongliangchang** [Kurumi233/Mutual-Channel-Loss#1]
The code is ok, and I will add a link  in the MC-Loss homepage to show your work.
Many Thanks.

**Comment by Kurumi233** [comment]
> The code is ok, and I will add a link in the MC-Loss homepage to show your work.
> Many Thanks.

My pleasure.

**Comment by dongliangchang** [comment]
Some setting advice,
1. Trained from scratch,  the alpha set to 2, we can get a better result. <Loss =ce_loss + 2.0 * (L_dis + 10.0 * L_div)>
2. Using Imagenet pretrained model, the batch_size set to 64, we can get a better result.

In addition to the above, there still have a "mistake", when we use 224 as input size, if you want to obtain the same results with paper report, you need remove the centercrop in test_transform. 
like this: 
```
transform_train = transforms.Compose([
    transforms.Scale((224,224)),
    transforms.RandomCrop(224, padding=4),
    transforms.RandomHorizontalFlip(),
    transforms.ToTensor(),
    transforms.Normalize(mean , std),
])

transform_test = transforms.Compose([
    transforms.Scale((224,224)),
    transforms.ToTensor(),
    transforms.Normalize(mean , std),
])
```

When we Using Imagenet pretrained model, your transform is right.  Sorry for this.

**Comment by Kurumi233** [comment]
> Some setting advice,
> 
> 1. Trained from scratch,  the alpha set to 2, we can get a better result. <Loss =ce_loss + 2.0 * (L_dis + 10.0 * L_div)>
> 2. Using Imagenet pretrained model, the batch_size set to 64, we can get a better result.
> 
> In addition to the above, there still have a "mistake", when we use 224 as input size, if you want to obtain the same results with paper report, you need remove the centercrop in test_transform. Sorry for this.

OK, I'll change the alpha and transforms. But I can't use a larger batchsize, because of the limitation of GPUs. It will take 8G each gpu when I use batchsize 32 and input size 448, with 2 RTX 2080 Ti GPUs. So I reduced the initial learning rate by half, maybe there will be a little impact to the results.

**Comment by dongliangchang** [comment]
> 
> 
> > Some setting advice,
> > 
> > 1. Trained from scratch,  the alpha set to 2, we can get a better result. <Loss =ce_loss + 2.0 * (L_dis + 10.0 * L_div)>
> > 2. Using Imagenet pretrained model, the batch_size set to 64, we can get a better result.
> > 
> > In addition to the above, there still have a "mistake", when we use 224 as input size, if you want to obtain the same results with paper report, you need remove the centercrop in test_transform. Sorry for this.
> 
> OK, I'll change the alpha and transforms. But I can't use a larger batchsize, because of the limitation of GPUs. It will take 8G each gpu when I use batchsize 32 and input size 448, with 2 RTX 2080 Ti GPUs. So I reduced the initial learning rate by half, maybe there will be a little impact to the results.

Batch size is important for fine-tune, a guy obtained 87.27 (Acc@1) at 150epoch using your code, just change the batch size from 32 to 64, with 4 RTX 2080 Ti GPUs.

**Comment by Kurumi233** [comment]
> > > Some setting advice,
> > > 
> > > 1. Trained from scratch,  the alpha set to 2, we can get a better result. <Loss =ce_loss + 2.0 * (L_dis + 10.0 * L_div)>
> > > 2. Using Imagenet pretrained model, the batch_size set to 64, we can get a better result.
> > > 
> > > In addition to the above, there still have a "mistake", when we use 224 as input size, if you want to obtain the same results with paper report, you need remove the centercrop in test_transform. Sorry for this.
> > 
> > 
> > OK, I'll change the alpha and transforms. But I can't use a larger batchsize, because of the limitation of GPUs. It will take 8G each gpu when I use batchsize 32 and input size 448, with 2 RTX 2080 Ti GPUs. So I reduced the initial learning rate by half, maybe there will be a little impact to the results.
> 
> Batch size is important for fine-tune, a guy obtained 87.27 (Acc@1) at 150epoch using your code, just change the batch size from 32 to 64, with 4 RTX 2080 Ti GPUs.

That's a good news (Envious eyes). Since the school server is closed, I can only use 2 GPUs.
